scripts/exportAssets.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+import os
+import sys
+import argparse
+import json
+from util import ElasticsearchUtil
+from util import Logger
+from util import Utility
+
+EXPORT_LOG = "/tmp/ExportAssets.log"
+
+esUtil = ElasticsearchUtil(EXPORT_LOG)
+logger = Logger(log_file=EXPORT_LOG)
+logging, rotating_handler = logger.configure_and_return_logging()
+UTIL = Utility(log_file=EXPORT_LOG)
+
+OUTPUT_DIR = os.path.dirname(os.path.realpath(__file__)) 
+
+INDEX = esUtil.KIBANA_INDEX
+TYPE = None
+ID = None
+
+DASHBOARD = "dashboard"
+VISUALIZATION = "visualization"
+SEARCH = "search"
+
+TO_FILE = {}
+
+
+def get_asset(es_index, es_type, es_id):
+   ignored, doc_exists = esUtil.function_with_timeout(esUtil.ES_QUERY_TIMEOUT,
+                                           esUtil.document_exists,
+                                             es_index,
+                                             es_type,
+                                             es_id)
+   if doc_exists:
+      success, ret = esUtil.get_document(es_index, es_type, es_id)
+      logging.info("GET RETURNS: " + "\n" + UTIL.pretty_format(ret))
+      return True, ret
+   else:
+      logging.info("No such thing as " + es_id + " in /" + es_index + "/" + es_type + "/")
+      return False, "Not found"
+
+def print_error_and_usage(argParser, error):
+   print "Error:  " + error + "\n"
+   print argParser.print_help()
+   sys.exit(2)
+
+def santize_input_args(arg_parser, args):
+   if len(sys.argv) == 1:
+      print_error_and_usage(argParser, "No arguments supplied.")
+   if (args.dash_name is None
+      and args.viz_name is None
+      and args.search_name is None):
+      print_error_and_usage(argParser, "Must have one of the following flags: -d -v -s")
+
+def get_filename(asset_title):
+   return asset_title + ".json"
+
+def get_full_path(asset_id):
+   global OUTPUT_DIR
+   if OUTPUT_DIR[-1:] == '/':
```

Comment:
I don't think you need the colon here. I think OUTPUT_DIR[-1] should work. You can ask Dan or Shweta, they would know better. But if you're trying to get the last element of the list, there shouldn't be a need to splice it.